packages/nextjs/components/scaffold-eth/NoContractAddressError.tsx

```diff
@@ -0,0 +1,13 @@
+import Address from "./Address";
+
+export default function NoContractAddressError({ address }: { address: string }) {
+  return (
+    <>
+      <Address address={address} />
+      <p className="break-normal my-1">
+        Above address dosen&apos;t seems like an contract address did you forgot to do
+        <br /> `yarn deploy` ?
+      </p>
+    </>
+  );
+}
```

Comment:
This is super specific for our use case (it's a toast error when using useScaffoldContractWrite). Not sure if we'll be using the component somewhere else, so let's not optimize early.

I think we can inline it in `useScaffoldContractWrite`: directly on the toast message, or in a component above.

Reply:
Ok, sorry I didn't read this!!!

> For this I have createad a new component inside packages/nextjs/components/scaffold-eth because tailwind classes dosen't work inside packages/nextjs/hooks/scaffold-eth since its path not included in tailwind.config.js.
> Should we inline it inside useScaffoldContractWrite and add path to tailwind.config.js ? Also I think current approach is not minimal of showing error with component so fee free to point it out and will add a simple message asap instead of creating new components raised_hands

Yesss, let's add it to the config.

Love to see that we share the same mindset.